cdn/single-origin.yml

```diff
@@ -347,10 +347,10 @@ Parameters:
       when the origin is an S3 bucket. Turning this on will not update any S3
       bucket policies, but such a policy is required to allow the origin access
       identity to function properly.
-    Default: 'No'
+    Default: "No"
     AllowedValues:
-      - 'Yes'
-      - 'No'
+      - "Yes"
+      - "No"
```

Comment:
Seems like prettier will change single -> double quotes, but won't go as far as removing unnecessary quotes?  So I guess that'll just be a stylistic thing for whomever is writing the template.

Reply:
In this case these quotes aren't unnecessary. `Yes` and `No` are different than `"Yes"` and `"No"` in YAML, since the former are the same as `true`, `on`, `y`, and all the other boolean literal values that exist in YAML.